security-analysis/security-analysis-api/src/main/java/com/powsybl/security/NodeBreakerViolationLocation.java

```diff
@@ -18,26 +18,18 @@ public class NodeBreakerViolationLocation implements ViolationLocation {
     private final List<String> busBarIds;
 
     public NodeBreakerViolationLocation(String voltageLevelId, List<String> busBarIds) {
-        Objects.requireNonNull(voltageLevelId, "voltageLevelId");
-        this.voltageLevelId = voltageLevelId;
-        this.busBarIds = busBarIds;
+        this.voltageLevelId = Objects.requireNonNull(voltageLevelId, "voltageLevelId should not be null");
```

Comment:
do not we remove the voltageLevelId in Location ?

Reply:
We could as it is redundant we busbar section Ids which allows to get the voltage level directly BUT 
 - using the busbar section is a design issue too
 - we should use nodes (integer) instead to covers all use cases and be symmetrical with what is in the  BusBreakerViolationLocation

So maybe we should keep it and in a next PR, remove busbar section and replace by nodes.

Reply:
Or maybe we should do it now?

Reply:
To avoid a painful migration